### DISCUSSION TITLE
Soundy v1.0.0

### DISCUSSION THREAD

**Discussion by zafercandarli** [Pandy999/Soundy#3] [General]
# **Soundy v1.0.0**
### Soundy is an AI chatbot that has different personalities and characteristics. 
- This is the first version, with some basic prompts, commands, events, etc. 
- Details can be found in the [wiki](https://github.com/Pandy999/Soundy/wiki).
- Official [Soundy Support Discord Server](https://discord.gg/zN67eGzxZC).
- Enjoy the bot :)

<hr /><em>This discussion was created from the release <a href='https://github.com/Pandy999/Soundy/releases/tag/v1.0.0'>Soundy v1.0.0</a>.</em>

**Comment by 4mlsx** [comment]
Does it need open ai api key?

**Comment by zafercandarli** [comment]
Hello,
yes, you need an api key.

**Reply by 4mlsx** [reply]
Where I need to put the api key

**Comment by zafercandarli** [comment]
Read the README file, it is all explained there.
